test/cli/rcmgr_test.go

```diff
@@ -63,12 +55,16 @@ func TestRcmgr(t *testing.T) {
 		})
 		node.StartDaemon()
 
-		res := node.RunIPFS("swarm", "limit", "system", "--enc=json")
+		res := node.RunIPFS("swarm", "resources", "--enc=json")
 		require.Equal(t, 0, res.ExitCode())
 		limits := unmarshalLimits(t, res.Stdout.Bytes())
 
-		assert.GreaterOrEqual(t, limits.ConnsInbound, 2000)
-		assert.GreaterOrEqual(t, limits.StreamsInbound, 2000)
+		if limits.System.ConnsInbound != rcmgr.Unlimited {
```

Comment:
Why do we have these `if` conditionals?  Can't we just do the `assert.GreaterOrEqual` as before?

Reply:
`rcmgr.Unlimited` is stored as `-1` which is smaller than `2000` even tho in our interpretation it is bigger than anything else.